packages/components/src/components/carousel.tsx

```diff
@@ -40,9 +40,10 @@ type Props = {
     gutter: number
     fetchGifs: (offset: number) => Promise<GifsResult>
     onGifsFetched?: (gifs: IGif[]) => void
+    noResultMessage?: string | HTMLElement
 } & EventProps
 
-const defaultProps = Object.freeze({ gutter: 6, user: {} })
+const defaultProps = Object.freeze({ gutter: 6, user: {}, noResultMessage: 'No results' })
```

Comment:
@JeffersonFilho do you think it should be `noResultsMessage`? instead of `noResultMessage`. Seeing `noResultMessage: 'No results'` here makes me lean towards the former. nbd

Reply:
@giannif Yeah probably make more sense we change to `noResultsMessage` instead of `noResultMessage` to keep things consistent.